ggml.h

```diff
@@ -205,6 +205,45 @@
         } \
     } while (0)
 
+#define GGML_RECOVERABLE_ERRORS
+#ifndef GGML_RECOVERABLE_ERRORS
+#define GGML_RECOVERABLE_ASSERT(_ctx, _code, x, ...) \
+    do { \
+        if (!(x)) { \
+            printf(__VA_ARGS__); \
+            fprintf(stderr, "GGML_ASSERT: %s:%d: %s\n", __FILE__, __LINE__, #x); \
```

Comment:
could these be defined as e.g. 

```suggestion
            fprintf(stderr, "GGML_ASSERT: %s:%d: %s: %s\n", __FILE__, __LINE__, __func__, #x); \
```

? That way, asserts wouldn’t have to handle the function name boilerplate.